sql/src/main/java/io/crate/analyze/symbol/InputColumn.java

```diff
@@ -101,6 +118,10 @@ public int index() {
         return index;
     }
 
+    private void index(int index) {
```

Comment:
since this is private can't you do the index assignment directly without function?